__test__/UserCollectableAttributes.test.js

```diff
@@ -317,4 +317,81 @@ describe('UCA Constructions tests', () => {
 
     expect(createUCA).toThrowError(`Version ${badVersion} is not supported for the identifier ${identifier}`);
   });
+
+  test('Construct cvc:socialSecurity:serialNumber successfully', () => {
+    const identifier = 'cvc:socialSecurity:number';
```

Comment:
socialSecurity should be SocialSecurity pascal case as are all the others

Reply:
Fixed.